jsk_fetch_robot/README.md

```diff
@@ -47,6 +47,8 @@ if [ $ROS_DISTRO = "indigo" ]; then
   wstool merge -t . https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.indigo
 elif [ $ROS_DISTRO = "kinetic" ]; then
   wstool merge -t . https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.kinetic
+elif [ $ROS_DISTRO = "melodic" ]; then
+  wstool merge -t . https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.melodic
```

Comment:
how about 
```
if [[ "$ROS_DISTRO" =~ ^(indigo|kinetic|meldoic)$ ]]; then
wstool merge -t . http......jsk_fetch_user.rosinstall.$ROS_DISTRO`
else
echo "..."
fi
```

Reply:
Thank you for your advice.
I updated the if-statement like your style.